prow/config.yaml

```diff
@@ -77,6 +77,48 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+    run_after_success:
+    - name: pull-kubernetes-e2e-kubeadm-gce
+      context: Jenkins kubeadm e2e
+      always_run: false
+      skip_report: true
+      rerun_command: "@k8s-bot kubeadm e2e test this"
+      trigger: "@k8s-bot (kubeadm e2e )?test this"
```

Comment:
Yeah, I'll lean on the EngProd team to advise on the safest way to roll this out, but my overall plan after discussion with @spxtr was going to be:

1) Start with this config so that the job runs nowhere by default. Then, I can create a fake PR and use the trigger command to manually have it run. This will help iron out any brand-new-job kinks (which there always seem to be, because there isn't support to test this configuration locally ahead of time).
2) Once the kinks are ironed out, add `run_if_changed` so it will always run and report results for kubeadm changes (but not block PRs).
3) Eventually, if the job proves stable enough, have failed runs block PRs.